algotrading/data/feature.py

```diff
@@ -1,26 +1,28 @@
-import pandas
+import pandas as pd
 from enum import Enum
 
-class Feature:
-    def __init__(self, name, variableType, transform):
+
+class Feature(object):
+    def __init__(self, name, variable_type, transform):
         self.name = name
-        self.variableType = variableType
+        self.variable_type = variable_type
         self.transform = transform
 
+    @staticmethod
```

Comment:
Any reason why a lot of your functions are disjoint from your classes? Seems like they are kind of related?

Reply:
functional programming style carry over.  im not a big fan of classes for much more than data types.

personally - i probably wouldn't use classes at all in general in python - they are handy for showing what an object should look like though :/

Reply:
hmm alright I'll change that back, I think it make the importing a little cleaner down the line